install_config/deploying_cfme.adoc

```diff
@@ -0,0 +1,510 @@
+
+Core variables in this role:
+
+[cols="3*", options="header"]
+|===
+|Name
+|Default Value
+|Description
+
+|`openshift_cfme_install_app`
+|`False`
+|`True`: Install everything and create a new CFME application. `False`: Only install the templates and scaffolding.
+|===
+
+Variables that you can override have defaults defined in
+ifdef::openshift-origin[]
+link:https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_cfme/defaults/main.yml[*_~/openshift-ansible/roles/openshift_cfme/defaults/main.yml_*].
+endif::[]
+ifdef::openshift-enterprise[]
+*_/usr/share/ansible/openshift-ansible/roles/openshift_Cfme/defaults/main.yml_*.
+endif::[]
+
+[[deploying-cfme-preflight-checks]]
+== Pre-flight Checks
+
+As documented in xref:deploying-cfme-prerequisites[Prerequisites], you must
+already have your {product-title} cluster up and running.
+
+The CFME pod is fairly large (about 1.7 GB), so to save some spin-up time
+post-deployment, you can optionally begin pre-pulling the following to each of
+your nodes now:
+
+----
+ifdef::openshift-origin[]
+# docker pull docker.io/manageiq/manageiq-pods:app-latest-fine
+endif::[]
+ifdef::openshift-enterprise[]
+# docker pull registry.access.redhat.com/cloudforms45/cfme-openshift-app
+# docker pull registry.access.redhat.com/cloudforms45/cfme-openshift-postgresql
+# docker pull registry.access.redhat.com/cloudforms45/cfme-openshift-memcached
+endif::[]
+----
+
+[[deploying-cfme-running-the-playbook]]
+== Running the Playbook
+
+[IMPORTANT]
+====
+Do not re-run the entrypoint playbook multiple times in a row without cleaning
+up after previous runs if some of the CFME steps have ran. This is a known
+issue. Further instructions are provided in
+xref:deploying-cfme-manual-cleanup[Manual Clean-up].
+====
+
+To run the playbook to deploy CFME: 
+
+. In your existing inventory file, set the `openshift_cfme_install_app` parameter
+to `True` under the `[OSEv3:vars]` section:
++
+----
+[OSEv3:vars]
+openshift_cfme_install_app=True
+----
+
+. Run the following entrypoint playbook to install CFME using your existing
+inventory file:
++
+----
+# ansible-playbook -v [-i /path/to/file] \
+ifdef::openshift-origin[]
+    ~/openshift-ansible/playbooks/byo/openshift-cfme/config.yml
+endif::[]
+ifdef::openshift-enterprise[]
+    /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cfme/config.yml
+endif::[]
+----
+
+[[deploying-cfme-verifying-the-deployment]]
+== Verifying the Deployment
+
+After the installation completes, the playbook shows the following information:
+
+----
+TASK [openshift_cfme : Status update] *********************************************************
+ok: [cfme-node.example.com] => {
+    "msg": "CFME has been deployed. Note that there will be a delay before it is fully initialized.\n"
+}
+----
+
+This will take several minutes (possibly 10 or more), depending on your network
+connection. However, you can get some insight into the deployment process during
+initialization.
```

Comment:
Removed it here and added it to the beginning of the next subsection:

"To gain further insight into the deployment process during initialization, use the `oc describe` command..."